src/pages/Popup/index.tsx

```diff
@@ -14,7 +14,7 @@ function Popup() {
     }, []);
 
     return (
-        <Layout.Container>
+        <Layout.Container fixedWidth>
```

Comment:
when is fixedWidth = false?

Reply:
When the popup opens in a window, user can change window size